security-analysis/security-analysis-api/src/test/java/com/powsybl/security/SecurityTest.java

```diff
@@ -125,14 +124,27 @@ void printLimitsViolations() {
     @Test
     void checkLimits() {
         List<LimitViolation> violations = Security.checkLimits(network);
-
-        assertEquals(5, violations.size());
```

Comment:
it does not pass ?

Reply:
yes it does, but it is also checked by assertViolations()